src/server/utils/wallets/getAwsKmsAccount.ts

```diff
@@ -0,0 +1,120 @@
+import { type KMSClientConfig } from "@aws-sdk/client-kms";
+import { KmsSigner } from "aws-kms-signer";
+import type { Hex, ThirdwebClient } from "thirdweb";
+import { keccak256, type Address as TwAddress } from "thirdweb";
+import { eth_sendRawTransaction, getRpcClient } from "thirdweb/rpc";
+import { serializeTransaction } from "thirdweb/transaction";
+import { toBytes } from "thirdweb/utils";
+import { type Account } from "thirdweb/wallets";
+import type {
+  SignableMessage,
+  TransactionSerializable,
+  TypedData,
+  TypedDataDefinition,
+} from "viem";
+import { hashTypedData } from "viem";
+import { getChain } from "../../../utils/chain";
+
+type SendTransactionResult = {
+  transactionHash: Hex;
+};
+
+type SendTransactionOption = TransactionSerializable & {
+  chainId: number;
+};
+
+type AwsKmsAccountOptions = {
+  keyId: string;
+  config?: KMSClientConfig;
+  client: ThirdwebClient;
+};
+
+export async function getAwsKmsAccount(
+  options: AwsKmsAccountOptions,
+): Promise<Account> {
+  const { keyId, config, client } = options;
+  const signer = new KmsSigner(keyId, config);
+
+  // Populate address immediately
+  const addressUnprefixed = await signer.getAddress();
```

Comment:
This seems weird the underlying lib returns the address unprefixed. Is that documented somewhere?

Reply:
don't think so, documentation is scarce, I mostly went by interface and source code